src/app/layout.js

```diff
@@ -16,7 +16,9 @@ const poppins = Poppins({
 export default function RootLayout({ children, session }) {
   return (
     <html lang="en" className="h-full">
-      <body className={`${poppins.variable} flex flex-col lg:flex-row h-full`}>
+      <body
+        className={`${poppins.variable} flex flex-col lg:flex-row h-full w-full`}
```

Comment:
yea It's hard to test this part without deploying but you can just add the bg-design-whichever green it is for the top color here next to w-full. It should work on mobile once I deploy it. I can check that later.